src/screens/Home/index.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+import Home from './index';
+
+const queryClient = new QueryClient();
+
+describe('Home component', () => {
+  test('matches last snapshot', () => {
+    const component = render(
+      <QueryClientProvider client={queryClient}>
+        <Home />
+      </QueryClientProvider>
+    );
+    expect(component).toMatchSnapshot();
```

Comment:
```suggestion
    expect(component).toMatchSnapshot();
    component.getByText(`${i18next.t('Global:loading')}...`);
```

you need to see if the home component renders well and for that you can check if it loads the text from **Global:loading**